### DISCUSSION TITLE
Modules in config?

### DISCUSSION THREAD

**Discussion by timtorres** [gethyas/hyas#214] [Q&A] [answered]
Hi, modules are mentioned in the [docs](https://gethyas.com/docs/recipes/project-configuration/#modules) but not in the repo config. Are the docs outdated? And what is being done here?
Thanks!

**Comment by h-enk** [comment, marked as answer]
Thanks! Yes, the docs are outdated. The module mounts* were removed for a (even) smarter solution with https://github.com/h-enk/hyas/pull/155

\* A way (using Hugo’s virtual filesystem) to access JavaScript files from node modules, needed for processing as asset files — see also the Hugo docs: [Module Config: mounts](https://gohugo.io/hugo-modules/configuration/#module-config-mounts)

I've raised an issue (so we don't forget): https://github.com/h-enk/hyas/issues/215

**Reply by h-enk** [reply]
FYI: [4b3178c048095722541153e4ee5b94b9cd3bc831](https://github.com/h-enk/gethyas.com/commit/4b3178c048095722541153e4ee5b94b9cd3bc831)
